### PR TITLE
Polkadot parachain node

### PR DESCRIPTION
**Pull request by agouin**
Phase 3 of #67 

Polkadot chains are composed of the relay chain (polkadot chain) nodes and the parachain nodes for involved parachains (e.g. Composable). This brings in the parachain node docker orchestration methods.

### CODE REVIEW

File: chain/polkadot/parachain_node.go [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/avast/retry-go/v4"
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	p2pCrypto "github.com/libp2p/go-libp2p-core/crypto"
```
Comment (mark-rushakoff) [inline chain/polkadot/parachain_node.go:17]: Import names should be all lowercase.

**Review by mark-rushakoff — APPROVED** [review]
A couple small style notes but otherwise looks ready to ship.

File: chain/polkadot/parachain_node.go (hunk 1 above)
Comment (DavidNix) [reply chain/polkadot/parachain_node.go:17]: You'd think a linter somewhere should catch that.

Comment (mark-rushakoff) [reply chain/polkadot/parachain_node.go:17]: Yeah, you'd think so. The blog post on this is 7.5 years old, so there's been plenty of time for someone to write one: https://go.dev/blog/package-names

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns...

Comment (DavidNix) [reply chain/polkadot/parachain_node.go:17]: There may be a linter with golangci-lint we need to activate. There's so many of them; hard to keep track.

**Pull request merged**